packages/back-end/src/models/TagModel.ts

```diff
@@ -27,16 +27,38 @@ function toTagInterface(doc: TagDocument | null): TagInterface[] {
       id: t,
       color: settings[t]?.color || "#029dd1",
       description: settings[t]?.description || "",
+      label: settings[t]?.label || t,
     };
   });
 }
 
+export async function getTag(
+  organization: string,
+  tag: string
+): Promise<TagInterface | null> {
+  const doc = await TagModel.findOne({
+    organization,
+  });
+  if (!doc) return null;
+
+  if (!doc.tags.includes(tag)) return null;
+
+  const settings = doc.settings || {};
+  return {
+    id: tag,
+    color: settings[tag]?.color || "#029dd1",
```

Comment:
This is duplicated above. It should this be factored out and placed as a constant on top of the file.